script/workLoggingSystem.js

```diff
@@ -125,6 +127,8 @@ $("#submitNewTask").click(function() {
     var estimatedTime = $("#estimatedTimeCreateNewTask").val();
     var status = $("#statusCreateNewTask").val();
     var category = $("#categoryCreateNewTask").val();
+    var d = new Date();
+    var dates = d.getDate() + "/" + (d.getMonth()+1) + "/" + d.getFullYear();
```

Comment:
I don't think there is again a need to create Date object. Please reuse it from id show_date

Reply:
i have tried this but this is not working.